src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', function () {
+  const table = document.querySelector('.dashboard');
+  // eslint-disable-next-line no-unused-vars
+  const row = table.querySelectorAll('tbody tr');
+
+  // eslint-disable-next-line no-undef, no-shadow
+  rows.forEach((row) => {
```

Comment:
It seems like the variable `rows` is not defined in the current scope. You should define or retrieve `rows` before using it in the `forEach` loop. Consider using `const rows = table.querySelectorAll('tbody tr');` instead of `const row` on line 6.